tests/test_deployment_model.py

```diff
@@ -0,0 +1,252 @@
+import os
```

Comment:
Need tests for the adaptors to verify they work on all the base models. If you add it to the test_base_models.py file this will achieve this.